test/tests/pbs_smoketest.py

```diff
@@ -558,6 +558,7 @@ def test_shrink_to_fit(self):
                 (GE, a['Resource_List.min_walltime'])}
         self.server.expect(JOB, attr, id=jid)
 
+    @skipOnCpuSet
```

Comment:
I would suggest changing test case not to set ncpus instead skipping it because the test is about a testing job with job script and it has nothing to do about how many ncpus we have (as long as we can run simple 1-ncpus job)